docs/.gitignore

```diff
@@ -18,6 +18,7 @@ report-check-links.xlsx
 /.vuepress/public/handsontable.css
 
 /next/api/
+/docker/redirects.conf
```

Comment:
This line was added into the middle `/next/api` ignore defs:
```
/next/api/
!/next/api/introduction.md
!/next/api/plugins.md
!/next/api/sidebar.js
```

I want to keep it in one piece. Could You please move this line above or after ?

Reply:
Done https://github.com/handsontable/handsontable/pull/8336/commits/1f604a8f1975eb7a1a201408f60f95f89c76f973